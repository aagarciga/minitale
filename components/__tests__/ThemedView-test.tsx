import React from "react";

import renderer from "react-test-renderer";

import { ThemedView } from "../ThemedView";

describe("ThemedView component", () => {
  it(`renders correctly`, () => {
    const tree = renderer
      .create(<ThemedView>Snapshot test!</ThemedView>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
