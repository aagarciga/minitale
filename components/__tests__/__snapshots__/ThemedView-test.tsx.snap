// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemedView component renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#F5F5F5",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#F5F5F5",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</View>
`;
